.github/workflows/publish.yml

```diff
@@ -0,0 +1,27 @@
+name: Publish
+
+on: push
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Build with bikeshed
+        run: |
+          git clone --depth 1 https://github.com/tabatkins/bikeshed.git
+          sudo pip3 install --editable ./bikeshed
```

Comment:
perhaps we should do `--user` in the end instead of `sudo` here and below

Reply:
I can give that a try, was also thinking of adding pip caching to speed things up further.